processes-services/src/test/java/org/exoplatform/processes/notification/plugin/RequestCommentPluginTest.java

```diff
@@ -43,20 +50,7 @@ public void setUp() {
         PowerMockito.mockStatic(NotificationUtils.class);
         PowerMockito.mockStatic(ExoContainerContext.class);
         when(ExoContainerContext.getService(IDGeneratorService.class)).thenReturn(null);
-    }
-
-    @Test
-    public void isValid() {
```

Comment:
since this check is removed! https://github.com/exoplatform/processes/pull/259/files#diff-40b95ff19b9ed70de5910ace5bd4c84884e1cebdbbb54541aaaa21db338be2c2L40-L41